qa/L0_sequence_batcher/test.sh

```diff
@@ -56,76 +62,146 @@ export CUDA_VISIBLE_DEVICES=0
+    elif [[ $BACKEND == "libtorch" ]]; then
+      MODELS="$MODELS $DATADIR/qa_ensemble_model_repository/qa_sequence_model_repository/*_libtorch_sequence_int32"
+    fi
+  fi
+done
+
+for MODEL in $MODELS; do
+    cp -r $MODEL models1/. && \
+        (cd models1/$(basename $MODEL) && \
             sed -i "s/^max_batch_size:.*/max_batch_size: 4/" config.pbtxt && \
             sed -i "s/kind: KIND_GPU/kind: KIND_GPU\\ncount: 1/" config.pbtxt && \
             sed -i "s/kind: KIND_CPU/kind: KIND_CPU\\ncount: 1/" config.pbtxt)
-    cp -r $m models2/. && \
-        (cd models2/$(basename $m) && \
+    cp -r $MODEL models2/. && \
+        (cd models2/$(basename $MODEL) && \
             sed -i "s/^max_batch_size:.*/max_batch_size: 2/" config.pbtxt && \
             sed -i "s/kind: KIND_GPU/kind: KIND_GPU\\ncount: 2/" config.pbtxt && \
             sed -i "s/kind: KIND_CPU/kind: KIND_CPU\\ncount: 2/" config.pbtxt)
-    cp -r $m models4/. && \
-        (cd models4/$(basename $m) && \
+    cp -r $MODEL models4/. && \
+        (cd models4/$(basename $MODEL) && \
             sed -i "s/^max_batch_size:.*/max_batch_size: 1/" config.pbtxt && \
             sed -i "s/kind: KIND_GPU/kind: KIND_GPU\\ncount: 4/" config.pbtxt && \
             sed -i "s/kind: KIND_CPU/kind: KIND_CPU\\ncount: 4/" config.pbtxt)
 done
 
-for m in \
-        $DATADIR/qa_sequence_model_repository/plan_nobatch_sequence_float32 \
-        $DATADIR/qa_sequence_model_repository/netdef_nobatch_sequence_int32 \
-        $DATADIR/qa_sequence_model_repository/graphdef_nobatch_sequence_object \
-        $DATADIR/qa_sequence_model_repository/graphdef_nobatch_sequence_int32 \
-        $DATADIR/qa_sequence_model_repository/savedmodel_nobatch_sequence_float32 \
-        $DATADIR/qa_sequence_model_repository/onnx_nobatch_sequence_int32 \
-        $DATADIR/qa_sequence_model_repository/libtorch_nobatch_sequence_int32 \
-        $DATADIR/qa_ensemble_model_repository/qa_sequence_model_repository/*_plan_nobatch_sequence_float32 \
-        $DATADIR/qa_ensemble_model_repository/qa_sequence_model_repository/*_netdef_nobatch_sequence_int32 \
-        $DATADIR/qa_ensemble_model_repository/qa_sequence_model_repository/*_graphdef_nobatch_sequence_object \
-        $DATADIR/qa_ensemble_model_repository/qa_sequence_model_repository/*_graphdef_nobatch_sequence_int32 \
-        $DATADIR/qa_ensemble_model_repository/qa_sequence_model_repository/*_savedmodel_nobatch_sequence_float32 \
-        $DATADIR/qa_ensemble_model_repository/qa_sequence_model_repository/*_onnx_nobatch_sequence_int32 \
-        $DATADIR/qa_ensemble_model_repository/qa_sequence_model_repository/*_libtorch_nobatch_sequence_int32 ; do
-    cp -r $m models0/. && \
-        (cd models0/$(basename $m) && \
+MODELS=""
+for BACKEND in $BACKENDS; do
+  if [[ $BACKEND == "plan" ]]; then
+    MODELS="$MODELS $DATADIR/qa_sequence_model_repository/plan_nobatch_sequence_float32"
```

Comment:
I think you can avoid the ifs by changing to `MODELS="$MODELS $DATADIR/qa_sequence_model_repository/$BACKEND_nobatch_sequence_float32"`, then you don't need to handle every backends.

Reply:
The datatypes for the backends are different. FP32 for savedmodel, int32 for netdef graphdef onnx libtorch and so on

Reply:
True. You can still simplify it though. If you look at `get_datatype` in the Python test, only three backends are the exceptions.

Reply:
Done